content/en/contribute/docs/style-guide.md

```diff
@@ -192,6 +192,29 @@ Documentation pages should be written in [Markdown notation](https://www.markdow
 | **bold** | `**bold**` | `<b>bold</b>` or `<strong>bold</strong>` |
 | table | `|...|` using [markdown tables](https://www.markdownguide.org/extended-syntax/#tables) | `<table><tr><td>...</td></tr></table>` | 
 
+### Tabular schedules
+
+Lists of supplement or vaccination schedules should be in markdown and use the green circle emoji (🟢). Leave cells empty when no action is needed. Add the `<div class="schedule-table"></div>` HTML before the table to make columns narrower. 
```

Comment:
This could be presented as more generic, and use vaccinations as an example.

```suggestion
Displaying the occurence of events over time in a workflow is often done using a table. To keep these consistent we recommend using the tabular schedule format. 

For example, here is a sample vaccination schedules:
<div class="schedule-table"></div>
|| 6 mo | 12 mo | 18 mo | 2 yr | 2.5 yr | 3 yr | 3.5 yr | 4 yr | 4.5 yr | 5 yr |
|------------|--|--|--|--|--|--|--|--|--|--|
| Deworming  | |   | 🟢 | 🟢 | 🟢 | 🟢 |  | 🟢 | 🟢 | 🟢 |
| Vitamin A  | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 |

To acheive this use a markdown table with the green circle emoji (🟢) to mark events, leaving cells empty when no action is needed. The HTML `<div class="schedule-table"></div>` is used before the markdown table so that built-in styling can be applied to the table. Here is the code for the above example:

```markdown
<div class="schedule-table"></div>
|| 6 mo | 12 mo | 18 mo | 2 yr | 2.5 yr | 3 yr | 3.5 yr | 4 yr | 4.5 yr | 5 yr |
|------------|--|--|--|--|--|--|--|--|--|--|
| Deworming  | |   | 🟢 | 🟢 | 🟢 | 🟢 |  | 🟢 | 🟢 | 🟢 |
| Vitamin A  | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 |
``` 
```

Reply:
Note that the closing markdown ` ``` ` is not properly shown in the suggested change.